kratos/linear_solvers/amgcl_ns_solver.h

```diff
@@ -419,6 +421,7 @@ class AMGCL_NS_Solver : public LinearSolver< TSparseSpaceType,
 
     double mTol;
     int mVerbosity;
+    std::string mPressureVarName;
```

Comment:
I would use a variable pointer, avoids to retreive the variable each time and is less memory than a string

Reply:
done